docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -42,87 +42,194 @@ Antenatal care attendance module
 1.0 Overview
 ++++++++++++
 
-This module determines whether or not a simulant attends an antenatal care visit according to their ANC visit propensity value.
+This module determines whether or not a simulant attends an antenatal care visit according to their ANC visit 
+propensity value and the timing of their ANC visit (i.e., during first trimester or later pregnancy). 
 
-2.0 Module Diagram and Data
-+++++++++++++++++++++++++++++++
+ANC visit timing is particularly relevant to both the hemoglobin and facility choice components of this simulation, so for more information 
+on how outputs from this module will be used, refer to :ref:`our hemoglobin module documentation <2024_vivarium_mncnh_portfolio_hemoglobin_module>`
+and :ref:`our facility choice module documentation <2024_vivarium_mncnh_portfolio_facility_choice_module>`.
 
-2.1 Module Diagram
+2.0 Module Input and Output Data
+++++++++++++++++++++++++++++++++
+
+2.1 Module Description 
 ----------------------
 
-.. image:: antenatal_care_module_diagram.png
+ANC attendance **for full term pregnancies** will be modeled as a single variable with 4 possible exposure options:
 
-2.2 Module Inputs
----------------------
+A. Attends ANC during first trimester AND later pregnancy
+B. Attends ANC during first trimester but NOT later pregnancy
+C. Attends ANC during later pregnancy but NOT first trimester
+D. Does not attend ANC at all during pregnancy
 
-.. list-table:: Module required inputs
+.. list-table:: ANC exposure options
   :header-rows: 1
 
-  * - Input
-    - Source module
-    - Application
-    - Note
-  * - ANC propensity value
-    - :ref:`Initial attributes <2024_vivarium_mncnh_portfolio_initial_attributes_module>`
-    - Used to determine answer to decision node #1
+  * - 
+    - Visit during late pregnancy
+    - No visit during late pregnancy
+  * - **Visit during first trimester**
+    - A
+    - B
+  * - **No visit during first trimester**
+    - C
+    - D
+
+The below table describes what propensity values to use for each exposure option outlined above.
+
+.. list-table:: ANC exposure propensity values for full term pregnancies
+  :header-rows: 1
+
+  * - ANC exposure option
+    - Description
+    - Propensity value
+    - Notes
+  * - A
+    - Attends ANC during first trimester AND later pregnancy
+    - ``min(ANCfirst, ANC4)``
```

Comment:
It's a bit confusing to start using the variables `ANCfirst`, `ANC4`, etc., without defining them first. I see the "Module required inputs" table below shows where to get these values -- perhaps the present table should be placed somewhere after that one?